src/doc/performance-tuning.md

```diff
@@ -257,6 +439,71 @@ root@c819e0f106c4:/# taskset -cp $$
 pid 1's current affinity list: 0-15
 ```
 
+## Application-specific performance notes
+
+### lwAFTR
+
+#### Avoid fragmentation
+
+Fragment reassembly is a costly operation.  Make sure that network
+MTUs are set such that fragmentation is rare.
+
```

Comment:
https://github.com/Igalia/snabb/blob/lwaftr/src/program/lwaftr/doc/README.configuration.md
lwaftr.conf example includes:
```
max_fragments_per_reassembly_packet = 1,
max_ipv6_reassembly_packets = 10,
max_ipv4_reassembly_packets = 10,
```

The counters docs, in turn, show how to use lwaftr query -l to get the actual memory use for the ctable buffers.

I'm ok with it not getting documented yet, but it is in our main lwaftr branch.